### ISSUE TITLE
Issue with q-toolbar in header slot with input field in view

### ISSUE THREAD

**Issue by jsfeldman** [quasarframework/quasar#363]
I've noticed a small issue where when you have a q-toolbar on your page and click an input field low on the page on chrome for android the input field doesn't scroll up to display above the keyboard that pops out, but stays hidden behind the keyboard.

Here's a rough skeleton of the HTML:
```
<q-layout>
  <div slot="header" class="toolbar">
    <q-toolbar-title :padding="1">
        Header
    </q-toolbar-title>
  </div>
  <div class="layout-view">
    <div class="layout-padding">
...
      <input>
...
    </div>
  </div>
</q-layout>
```

I was able to fix it somewhat by editing the CSS a bit like so:
```
layout.layout-view.layout {
  bottom: auto;
  height: auto;
}
div.toolbar {
  position: fixed;
  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
}
.layout-view.layout-padding {
  margin-top: 45px; /* Roughly height of toolbar */
}
```
But I've noticed changing the CSS this way causes issues in safari on iPhones.

Has anyone else noticed this issue? Is it already known?

Let me know if you need any other information on the issue. I wish I could take a gif of it occurring to illustrate it better.

**Comment by rstoenescu** [comment]
This is not an easy issue due to how Cordova and the keyboard interact. Will make a full description when I have some more time.

**Comment by darrenahunter** [comment]
from memory I believe this is why the ionic guys wrote the cordova keyboard plugin https://github.com/driftyco/ionic-plugin-keyboard 
it may help here - gives more control over the popup keyboard

**Comment by rstoenescu** [comment]
Fixed in future v0.14 as part of Layout changes.
